docs/synthetic-tokens/long-short-pair.md

```diff
@@ -53,32 +50,69 @@ The method are:
+
+Regardless of the value of the long and short tokens when redeemed, the summed value of the two tokens will always be worth the total amount of collateral used to mint. The contract burns the tokens and returns collateral proportional to the amount that the token sponsor has deposited to the contract. The caller does not need to approve this contract to transfer any number of redeemed tokens since long and short tokens are burned, rather than transferred.
+
+![](/docs/lsp-tokens/lsp_redeem.png)
 
 ### `expire`
 
-After the expiration timestamp, anyone can call `expire` to request a price from the UMA Optimistic Oracle. This function takes no parameters and will revert if the current time is not later than `expirationTimestamp`. 
+When the current timestamp is later than the `expirationTimestamp` parameter, token holders are unable to settle their tokens for collateral until a price has been received by the Optimistic Oracle. The `settle` function will revert until the `expire` function is called once by anyone. 
+
+The `expire` function does not take any parameters and requests a price from the Optimistic Oracle for the LSP contract's `priceIdentifier`, `expirationTimestamp`, `customAncillaryData`, `collateralToken`, and `prepaidProposerReward`. At this time, a proposal bond can be set associated with the price request using `setBond` along with a custom liveness value for the request using `setCustomLiveness`. Liveness is the amount of time a proposal must wait before being resolved.
+
+![](/docs/lsp-tokens/lsp_expire.png)
 
 ### `settle`
 
-Once the Optimistic Oracle or DVM returns a price, token holders of either long or short tokens can call `settle` to burn tokens in return for collateral. The amount of collateral for each token is computed with `expiryPercentLong`.
+Once a price request exists, Proposers respond by referencing off-chain price feeds and submitting parameters for the `priceIdentifier`, `timestamp`, `ancillaryData`, and `proposedPrice` through the Optimistic Oracle contract. In return, Proposers receive a pre-defined proposal reward set by the Requestor. To propose prices, the Proposer is required to stake a proposal bond. If the price information they provided is disputed and deemed incorrect, the Proposer will lose their bond.
 
-## Financial Product Libraries (FPL)
+Disputers can refute a price submitted by a Proposer within the proposal liveness period by referencing their own off-chain price feeds. The proposal liveness period is a pre-defined amount of time a proposal can be disputed before the Requestor receives the price of the asset. If Disputers do not refute the price submitted by the Proposer within the proposal liveness period, the price is sent to the Requestor. If a proposal is disputed, the price will be submitted to UMA’s DVM and resolved after 48-hours.
```

Comment:
```suggestion
Disputers can refute a price submitted by a Proposer within the proposal liveness period by referencing their own off-chain price feeds. The proposal liveness period is a pre-defined amount of time a proposal can be disputed before the Requestor receives the price of the asset. If Disputers do not refute the price submitted by the Proposer within the proposal liveness period, the price is treated as correct and can now be read from the Optimistic Oracle. If a proposal is disputed, the price will be escalated to UMA’s Decentralized Verification Mechanism (DVM) and resolved after a 48-96 voting period.
```

Reply:
Make an update but want to confirm, shouldn't it be...after a 48-96 hour voting period?

Reply:
Yes correct. Isn't that what you have now?